cloud/google/gce_tag.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Do you mean to assign copyright to Google? You can/should associate this with yourself or your employer (if required).